### ISSUE TITLE
Adding New Coordinate Systems to the Map Location Feature

### ISSUE THREAD

**Issue by garciadb2187** [Esri/geoform-template-js#516]
Sir, I wanted to know if there is a possibility of having the option to move the map to the top of the map allowing this to be one of the first thing a user would see when opening the geoform. I also wanted to know if there could be addition to the coordinate system conversation that the map allows you to input ddd mm.mmm, ddd mm ddd, ddd mm ss.

**Comment by driskull** [comment]
Yes, we'll add this option when we make the 4x JSAPI version of this app.
